lms/djangoapps/shoppingcart/models.py

```diff
@@ -824,6 +824,69 @@ def generate_pdf_invoice(self, course, course_price, quantity, sale_price):
 
         return pdf_buffer
 
+    def __unicode__(self):
+        return "company: {} , internal reference: {} , customer reference number: {}".format(
+            self.company_name,
+            self.internal_reference,
+            self.customer_reference_number
+        )
+
+INVOICE_TRANSACTION_STATUSES = (
+
+    ('started', 'started'),
+    ('completed', 'completed'),
+    ('cancelled', 'cancelled'),
+    ('payment', 'payment'),
+    ('refund', 'refund'),
```

Comment:
We can determine whether the transaction is a payment or a refund by checking the sign of "amount".  I think you can remove "payment" and "refund" from this list.